samples/mongobetween_sample/README.md

```diff
@@ -0,0 +1,238 @@
+
+* An Amazon DocumentDB cluster with at least two db.r6g.large instances. You can use an existing Amazon DocumentDB cluster or [create a new one](https://docs.aws.amazon.com/documentdb/latest/developerguide/db-cluster-create.html). This post assumes the default values for port (27017) and TLS (enabled) settings.
+
+* A security group that enables you to connect to your Amazon DocumentDB cluster from your Amazon EC2 instance. You can use an existing security group or [create a new one](https://docs.aws.amazon.com/documentdb/latest/developerguide/get-started-guide.html#cloud9-security). You may also use the [Connect using Amazon EC2](https://docs.aws.amazon.com/documentdb/latest/developerguide/connect-ec2.html) feature to connect your Amazon DocumentDB cluster to your Amazon EC2 instance. 
+
+* Python 3+. Modules pymongo
+
+```
+pip install pymongo
+```
+
+## Setup Mongobetween  in the  Amazon EC2 Instance
+
+1.Install and Setup Mongobetween
+
+* SSH into your EC2 instance
+* Copy the files [```install_mongobetween.sh```](install_monogbetween.sh),[```mongo.go```](mongo.go),[```operations.go```](operations.go) to the EC2 instance home directory.The go files are needed to make the current main branch of Mongobetween( as of [commit](https://github.com/coinbase/mongobetween/commit/ca3d8d78d99847afc747b56c5b4ea31b85bff013)) to work with read preferences passed in connection URI string.
+* Run the following command to make the ```install_mongobetween.sh``` file executable: 
+
+        chmod 700 install_mongobetween.sh
+
+* Execute the script - this may take a while.
+
+        ./install_mongobetween.sh
+
+2.Add ```mongobetween``` executable to the path.
+
+Open the ```~/.bashrc``` file in your editor of choice, add the following lines to the end of the file, and save it.
+
+
+    export GOPATH="$HOME/go"
+    PATH="$GOPATH/bin:$PATH"
+
+
+3.Check if Mongobetween has been installed. 
+
+* Log out of your ec2 terminal and SSH back in.
+* Execute the  command ```mongobetween -h``` and check if you see the follwing output
+```
+    mongobetween -h
+    Usage: mongobetween [OPTIONS] address1=uri1 [address2=uri2] ...
+    -dynamic string
+            File or URL to query for dynamic configuration
+    -enable-sdam-logging
+            Enable SDAM(Server Discovery And Monitoring) logging
+    -enable-sdam-metrics
+            Enable SDAM(Server Discovery And Monitoring) metrics
+    -loglevel string
+            One of: debug, info, warn, error, dpanic, panic, fatal (default "info")
+    -network string
+            One of: tcp, tcp4, tcp6, unix or unixpacket (default "tcp4")
+    -password string
+            MongoDB password
+    -ping
+            Ping downstream MongoDB before listening
+    -pretty
+            Pretty print logging
+    -statsd string
+            Statsd address (default "localhost:8125")
+    -unlink
+            Unlink existing unix sockets before listening
+    -username string
+            MongoDB username
```

Comment:
DocumentDB?

Reply:
this is message mongobetween prints. to change this we need to change mongobetween code